Makefile

```diff
@@ -100,6 +101,35 @@ test-postgres-docker:
 		docker-compose -f $(CURDIR)/ppl/zqd/scripts/dkc-services.yaml down || exit; \
 		exit $$status
 
+.PHONY: test-temporal
+test-temporal: build bin/tctl
+	@ZTEST_PATH="$(CURDIR)/dist:$(CURDIR)/bin" \
+		ZTEST_TAG=temporal \
+		go test -v -run TestZq/ppl/zqd/temporal/ztests .
+
+.PHONY: bin/tctl
+bin/tctl: bin/tctl-$(TEMPORAL_TAG)
+	ln -fs $(<F) $@
+
+bin/tctl-$(TEMPORAL_TAG):
+	mkdir -p $(@D)
+	echo 'module deps' > $@.mod
+	go get -d -modfile=$@.mod go.temporal.io/server/cmd/tools/cli@$(TEMPORAL_TAG)
+	go build -modfile=$@.mod -o $@ go.temporal.io/server/cmd/tools/cli
+
+.PHONY: test-temporal-docker
+test-temporal-docker: bin/docker-compose.temporal-$(TEMPORAL_TAG).yaml
```

Comment:
Like I did in my [redis pr](https://github.com/brimsec/zq/pull/2040) I think you should merge this with test-postgres-docker to renamed test-services-docker and add the docker compose temporal stuff to ppl/zqd/scripts/dkc-services-yaml. I looked at the temporal docker-compose and it has stuff like temporal-admin-tools and temporal-web that we won't need for testing environments. Plus it would be nice to have temporal using the same postgres db that standup for postgres tests.

Reply:
I like that general direction, but noticed @nwt 's comment about temporal sometimes taking 10s to come up.

Reply:
Done in 07e423bb3547fe9a358c1fb14c5c09238d70305e.